src/components/molecules/TableComponent/TableComponent.scss

```
@@ -0,0 +1,99 @@
+@use "scss/attendee/mixins.scss"as *;
+@use "scss/attendee/variables.scss"as *;
```

Weird spacing going on.
```suggestion
@use "scss/attendee/mixins.scss" as *;
@use "scss/attendee/variables.scss" as *;
```